dagstore.go

```diff
@@ -210,6 +211,7 @@ func NewDAGStore(cfg Config) (*DAGStore, error) {
 		internalCh:        make(chan *task, 1),       // len=1, because eventloop will only ever stage another internal event.
 		completionCh:      make(chan *task, 64),      // len=64, hitting this limit will just make async tasks wait.
 		dispatchResultsCh: make(chan *dispatch, 128), // len=128, same as externalCh.
+		gcCh:              make(chan chan *GCResult, 8),
```

Comment:
Are we just going ahead with a random number here ?

Reply:
Yep, if the channel is backlogged, the next GC request will make the GC() call wait.